whylog/tests/tests_client/test_backtrackerSearcher.py

```diff
@@ -36,12 +36,20 @@ def _verify_lines(self, lines_normally, lines_reversed):
             assert i == j
 
     def _run_reverse_and_check_results(
-        self, log_file_path, reverse_from_offset_params, how_many_last_lines
+        self,
+        log_file_path,
+        reverse_from_offset_params,
+        how_many_last_lines,
+        final_offset_should_be_zero=True
     ):
         backtracker = searchers.BacktrackSearcher(log_file_path)
 
         lines = self._read_last_n_lines_from_file(log_file_path, how_many_last_lines)
-        lines_reversed = list(backtracker._reverse_from_offset(*reverse_from_offset_params))
+        data_reversed = list(backtracker._reverse_from_offset(*reverse_from_offset_params))
```

Comment:
casting to list here is not necessary - it is better to let the list comprehension in the next line pull from the generator